README.md

```diff
@@ -22,17 +22,20 @@ yarn add wolox-equalizer
 
 ## Usage
 
-1- Include the equalizer.scss before the index file of your CSS/SCSS.
+Include the equalizer.scss before **any** file of your CSS/SCSS. You can do it by simply using the import statement on top of your root `scss` file:
 
-If you want to import the 'equalizer' in this way:
+```scss
+@import '~wolox-equalizer/equalizer';
+```
 
-```js
+Or you can also do the following trick to simply import it with it's name:
+
+```scss
 @import 'equalizer';
 ```
 
-You have to add the following lines to the webpack configuration file: 
```

Comment:
I think this line should stay 'cause not always you have a `webpack.config.js` file.